### DISCUSSION TITLE
`token on mail not received`

### DISCUSSION THREAD

**Discussion by rajanshresth** [typehero/typehero#1153] [Q&A]
# It has been 9 days, I haven't get my early token on mail.
# How long do I need to wait for the early access token?
<img width="869" alt="image" src="https://github.com/typehero/typehero/assets/76089000/ac3171a0-bb9e-4ed1-829b-81b1e3e4a644">

**Comment by bautistaaa** [comment]
we send 500 tokens out a week randomly. github issues is not a place for this. bring it up in discord
